tools/rapids-upload-conda-to-s3

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# A utility script that tars up `/tmp/conda-bld-output` and uploads it to S3
+# Positional Arguments:
+#   1) a string of "cpp" or "python" which determines which conda artifact
+#      should be uploaded
+set -e
+
+PKG_TYPE="$1"
+case "${PKG_TYPE}" in
+  cpp)
+    ;&
+  python)
+    ;;
+  *)
+    echo 'Pass "cpp" or "python" as an argument.'
+    exit 1
+    ;;
+esac
+
+# Create tarball
+TAR_FILE="/tmp/${PKG_TYPE}.tar.gz"
+tar -czf "${TAR_FILE}" -C /tmp/conda-bld-output .
```

Comment:
I see what you mean now. This feature could be used in case someone wants to upload the conda workspace directory (i.e. `/tmp/conda-bld-workspace`), which includes all of the object files and CMake configs.

So it could be something like:

```sh
tar -czf "${TAR_FILE}" -C "${UPLOAD_DIR:-/tmp/conda-bld-output}" .
```